### DISCUSSION TITLE
Don't have access to GitHub Copilot

### DISCUSSION THREAD

**Discussion by linjingjin2019** [community/community#130464] [GitHub Education]
### Hi everyone,

I had my education pack approved. And following the Student Developer Pack offers to https://education.github.com/experiences/primer_copilot, then https://classroom.github.com/assignment-invitations/0499fb194a9671d96dbbeb1c95def64a/status. Following instruction to create the 
Codespaces within the repository and get the access error in the screenshot. 

Can someone please help me on this issue? I want to get co-pilot added to my  VS code.Thanks! 
<img width="1919" alt="Copilot Error" src="https://github.com/community/community/assets/140136623/fcdcfeee-c524-4d9b-89dc-c17d63f666b9">

**Comment by V4nshaj** [comment]
Use Tabnine AI or try uninstalling the github copilot and reinstall it

**Comment by fabifont** [comment]
I have the same problem. I have been approved but I can't use copilot. If I go to the main page I can only start a free trial even If I am a verified student.

Edit: solved by manually applying here https://github.com/github-copilot/free_signup
